ci/_init.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+export PLATFORM=$(echo $(uname) | tr '[:upper:]' '[:lower:]')
+export INTERNAL_REPO=nexus.int.snowflakecomputing.com:8086
```

Comment:
please checkout first SNOW-1042723 and CASEC-9266

Reply:
is there some timeline for CASEC-9266 ? I guess I can wait for decision but I rather not wait too long with merge